test/request.js

```diff
@@ -19,10 +19,10 @@ var assert = require('assert');
 var nock = require('nock');
 var request = require('../includes/request.js');
 
-describe('#unit Check request response', function() {
-  const url = 'http://localhost:5984';
-  const client = request.client(url, 1);
+const url = 'http://localhost:5984';
```

Comment:
Could be, changed in 35a873b3ea83c20731f1ffee0891391fa74c7e97
Also added a `beforeEach` to clean the mocks which I think will help for failure cases as I did in the fatalerrors test.